libsolidity/lsp/GotoDefinition.cpp

```diff
@@ -0,0 +1,79 @@
+#include <libsolidity/lsp/GotoDefinition.h>
+#include <libsolidity/lsp/Transport.h> // for RequestError
+#include <libsolidity/lsp/Utils.h>
+#include <libsolidity/ast/AST.h>
+#include <libsolidity/ast/ASTUtils.h>
+
+#include <fmt/format.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+using std::make_shared;
```

Comment:
Did we even run into problems recently where we had a name clash? I really don't like having to modify the `using` statements here whenever the code below changes. If all `move`s are gone, do you really think to remember removing this statement below?